### PR TITLE
feat(testcafe): add the support for testcafe

### PR DESCRIPTION
**Pull request by guiyomh**
This PR adding the support of TestCafe

### CODE REVIEW

**Comment by puneet0191** [comment]
Thanks a lot for your help with this @guiyomh, this will be reviewed and merged soon.

File: index.js [hunk 1]
```diff
@@ -369,17 +369,24 @@ class ResembleHelper extends Helper {
...
+      if (await els.count != 1) throw new Error(`Element ${selector} couldn't be located or isn't unique on the page`);
+    }
+    else {
+      if (!els.length) throw new Error(`Element ${selector} couldn't be located`);
+    }
 
     let location, size;
 
     if (this.helpers['Puppeteer']) {
+      const el = els[0];
```
Comment (puneet0191) [inline index.js:383]: what is the need for this?

File: index.js [hunk 2]
```diff
@@ -369,17 +369,24 @@ class ResembleHelper extends Helper {
...
       const box = await el.boundingBox();
       size = location = box;
     }
 
     if (this.helpers['WebDriver'] || this.helpers['Appium']) {
+      const el = els[0];
```
Comment (puneet0191) [inline index.js:389]: also here?

File: index.js (hunk 1 above)
Comment (guiyomh) [reply index.js:383]: I moved this assignment from line 373 here. Because that doesn't work like this for TestCafe.

File: index.js (hunk 2 above)
Comment (guiyomh) [reply index.js:389]: yes for the same reason. I didn't like this duplicate of code. But If you have a better solution, I'm ready to implement it.

**Comment by puneet0191** [comment]
@DavertMik could we merge this one?

**Comment by puneet0191** [comment]
@jonatask help me with some testing for this?

**Pull request merged**